.circleci/config.yml

```diff
@@ -54,15 +55,18 @@ jobs:
             curl -L "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl" -o /usr/local/bin/kubectl
             chmod +x /usr/local/bin/kubectl
             cp $(pwd)/artifacts/bin/okteto-Linux-x86_64 /usr/local/bin/okteto
+            /usr/local/bin/okteto login --token ${OKTETO_TOKEN}
       - run: 
           name: integration tests (serverside)
           command: |
             unset OKTETO_CLIENTSIDE_TRANSLATION
+            unset OKTETO_TOKEN
             make integration
       - run: 
           name: integration tests (clientside)
           command: |
             export OKTETO_CLIENTSIDE_TRANSLATION=true
+            unset OKTETO_TOKEN
```

Comment:
once we merge this PR I'll go ahead and remove it from here and from the project. They are here for now, just to make CI is not broken while this PR is in flux.